src/admin/js/Layout.js

```diff
@@ -8,7 +8,9 @@ import Notification from 'admin-on-rest/lib/mui/layout/Notification';
 import LayoutTitle from './LayoutTitle';
 
 const Layout = ({ isLoading, children }) => {
-    const RightElement = isLoading ? <CircularProgress color="#fff" size={0.5} /> : <span />;
+    const RightElement = isLoading
+        ? <CircularProgress color="#fff" size={30} thickness={2} style={{ margin: 8 }} />
+        : <span />;
```

Comment:
No, that's the way it is done in admin-on-rest currently. I haven't tested it but I suspect material-ui to have a funny behavior without it